docs/docs/Deployment/deployment-public-server.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Deploy your Langflow server using ngrok
+slug: /deployment-public-server
+---
+
+By default, your Langflow server at `http://localhost:7860` isn't exposed to the public internet.
+However, you can forward Langflow server traffic with a forwarding platform like [ngrok](https://ngrok.com/docs/getting-started/) or [zrok](https://docs.zrok.io/docs/getting-started) to make your server public.
+
+This allows you to [deploy your MCP server externally](#deploy-your-mcp-server-externally), [serve API requests](#serve-api-requests), and [share your playground externally](#share-your-playground-externally).
+
+The following procedure uses ngrok, but you can use any similar reverse proxy or forwarding platform.
+This procedure also assumes that you're using the default Langflow listening address `http://localhost:7860`.
+
+## Prerequisites
+
+- [Install Langflow](/get-started-installation)
+- [Install ngrok](https://ngrok.com/docs/getting-started/#1-install-ngrok)
+- [Create an ngrok authtoken](https://dashboard.ngrok.com/get-started/your-authtoken)
+
+## Expose your Langflow server with ngrok
+
+1. Start Langflow.
+
+    ```bash
+    uv run langflow run
+    ```
+
+2. In another terminal window, copy your [ngrok authtoken](https://dashboard.ngrok.com/get-started/your-authtoken) and use it to authenticate your local ngrok server:
```

Comment:
```suggestion
2. In another terminal window, use your ngrok authtoken to authenticate your local ngrok server:
```

Are they already supposed to have set up an ngrok server by this point? Or does this command start the server setup process?